scalableminds/webknossos
# Enforce magnifications via task type

@youri-k Can you update the migration guide? A datastore update is probably not necessary, is it?

Additionally, to the mag enforcement, I also added an "active resolution" hint to the info tab. I think, this is quite useful (at least, it was for testing :)).

### URL of deployed dev instance (used for testing):
- https://enforcemags.webknossos.xyz

### Steps to test:
- edit a task type and check "enforce magnifications"
- fill min and max (you can also try to omit one value, as this should work, too)
- create a task for that task type
- trace the task and ensure that you can only zoom within that range

### Issues:
- fixes #4075 

------
